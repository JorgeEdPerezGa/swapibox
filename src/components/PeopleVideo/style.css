.background-video {
  opacity: 0.4;
  width: 100%;
  margin-top: -30px; }
