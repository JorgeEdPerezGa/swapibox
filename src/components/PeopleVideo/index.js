import React from 'react';
import video from '../../assets/people.mp4';
import './style.css';

const PeopleVideo = () => {
  return (
    <div>
      <video
        poster = { video }
        className = 'background-video'
        playsInline
        autoPlay
        muted
        loop >
        <source
          src = { video }
          type = 'video/mp4'>
        </source>
      </video>
    </div>
  );
};

export default PeopleVideo;
