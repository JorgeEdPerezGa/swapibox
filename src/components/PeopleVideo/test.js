import React from 'react';
import PeopleVideo from './index';
import setupTest from '../../setupTests';
import { shallow } from 'enzyme';

describe('<PeopleVideo/>', () => {
  const wrapper = shallow(<PeopleVideo />);

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
