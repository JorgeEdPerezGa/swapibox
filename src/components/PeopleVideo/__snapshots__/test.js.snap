// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PeopleVideo/> should match snapshot 1`] = `<PeopleVideo />`;
